script/publish_persistent_udfs

```diff
@@ -67,12 +94,27 @@ def publish_persistent_udf(raw_udf, dataset, project_id):
         query_with_renamed_udfs = UDF_RE.sub(
             "`" + project_id + "`." + dataset + "." + r"\1", definition
         )
-        query = query_with_renamed_udfs.replace(
-            "CREATE TEMP FUNCTION", "CREATE OR REPLACE FUNCTION"
+
+        # adjust paths for dependencies stored in GCS
+        query = OPTIONS_LIB_RE.sub(
+            'library = "gs://' + gcs_bucket + "/" + gcs_path + r"\1" + '"',
+            query_with_renamed_udfs,
```

Comment:
I thought it might make sense to adjust the gcs paths of the external dependencies in the UDFs. Though it might be a unintuitive?

Reply:
Given that you have the gcs bucket as a configurable parameter, it makes sense that you'd adjust the paths to match. It's complexity that doesn't seem obviously necessary at this point, but given that the code is already written, it seems okay. For now, I would indeed like the paths in the UDF definitions to match exactly the path where we intend to deploy them.